src/components/content/catalog/services/details/ServiceDetail.tsx

```diff
@@ -3,7 +3,7 @@
  * SPDX-FileCopyrightText: Huawei Inc.
```

Comment:
also change the logo file name to ansible_logo.

Also use a full logo and reduce the size similar to deployer logo. 

Eg. https://www.google.com/url?sa=i&url=https%3A%2F%2Ficon-icons.com%2Ficon%2Fansible-logo%2F169596&psig=AOvVaw13zTc2GTlGPFKC0R97VotV&ust=1730199872668000&source=images&cd=vfe&opi=89978449&ved=0CBEQjRxqFwoTCPjcm_n2sIkDFQAAAAAdAAAAABAE